src/Sonarr.Api.V3/Series/SeriesController.cs

```diff
@@ -167,11 +166,8 @@ public ActionResult<SeriesResource> UpdateSeries(SeriesResource seriesResource)
         }
 
         [RestDeleteById]
-        public void DeleteSeries(int id)
+        public void DeleteSeries(int id, bool deleteFiles = false, bool addImportListExclusion = false)
```

Comment:
Does this need `[FromQuery]` to ensure it comes from the query string?

Reply:
Yeah, it definitely works without, seems like it can be used for more complex model binding https://learn.microsoft.com/en-us/aspnet/core/mvc/models/model-binding?view=aspnetcore-7.0